sdks/universal-router-sdk/src/entities/actions/uniswap.ts

```diff
@@ -118,15 +126,7 @@ export class UniswapTrade implements Command {
           addV3Swap(planner, swap, this.trade.tradeType, this.options, this.payerIsUser, routerMustCustody)
           break
         case Protocol.V4:
-          addV4Swap(
-            planner,
-            swap,
-            this.trade.tradeType,
-            this.options,
-            this.payerIsUser,
-            routerMustCustody,
-            performAggregatedSlippageCheck
```

Comment:
can you educate me what is `performAggregatedSlippageCheck` is and why is it being removed?

Reply:
yah I added that in my first PR and it's a bummer bc then `addV4Swap` has a different parameter interface than all the other `add<version>Swap` functions. But I can just surmise from "routerMustCustody" and tradetype whether it's necessary to perform the slippageCheck. Hence, I can do it from within the addV4Swap function with the information I'm given instead of outside of it.